docs/debugging.md

```diff
@@ -0,0 +1,31 @@
+# Debugging
+
+## Comparing with Cairo-Lang
+
+If you executed a Cairo0 proof program with both Rust and Python VM, you can use the following scripts to compare their output. They all require `delta` (modern diff) to be installed. If you don't have you can locally change it.
+
+No output when running a differ script implies that there are no differences.
+
+To compare the public inputs, run:
+```bash
+scripts/air_public_inputs_differ.bash file1 file2
```

Comment:
Can you be a little clearer here.
Instead of file do <`AIR-PUBLIC-INPUT-1`>  <`AIR-PUBLIC-INPUT-2`>

Try to do the same to the other commands

Reply:
Done